buildenv

```diff
@@ -63,6 +66,6 @@ unset GIT_EXEC_PATH
 unset GIT_MAN_PATH
 unset GIT_ROOT
 unset GIT_SHELL
-unset GIT_TEMPLATE_DIR
+export GIT_TEMPLATE_DIR="\$PWD/share/git-core/templates"
```

Comment:
is this required ?

Reply:
yes, when the install is relocated (e.g. when it's downloaded to the client's filesystem)